### ISSUE TITLE
Create buidler-js-plugin-boilerplate

### ISSUE THREAD

**Issue by alcuadrado** [nomiclabs/buidler#227]
We should have a JavaScript equivalent of https://github.com/nomiclabs/buidler-ts-plugin-boilerplate

**Comment by Palmatum** [comment]
Hey @alcuadrado, I would like to contribute

**Comment by alcuadrado** [comment]
Awesome! Do you wanna give it a try in a repository of yours and ping us back? We can then move it to nomiclabs/ for more visibility.

**Comment by Palmatum** [comment]
Yes I'm done with all the scripts and tests. All that is left now is commenting stuff and things like readme.
Here's a link of the [repository](https://github.com/Palmatum/hardhat-js-plugin-boilerplate).

**Comment by alcuadrado** [comment]
Nice! Looks good.

> All that is left now is commenting stuff and things like readme.

Let us know when this is ready :)

**Comment by Palmatum** [comment]
Sure I'll let you know when it's done :)

**Comment by Palmatum** [comment]
Hey! I'm done with the boilerplate, you can check it here:
[repository](https://github.com/Palmatum/hardhat-js-plugin-boilerplate).

**Comment by Palmatum** [comment]
Hey @alcuadrado ! I'm done with the boilerplate, you can check it out [here](https://github.com/Palmatum/hardhat-js-plugin-boilerplate).

**Comment by Palmatum** [comment]
> Hey @alcuadrado ! I'm done with the boilerplate, you can check it out [here](https://github.com/Palmatum/hardhat-js-plugin-boilerplate).


Hey @alcuadrado @fvictorio is there any feedback you want to give? As I'm new to contributing into projects and Hardhat is my first contribution. So I don't really know what I did wrong or right. Is there any feedback by your end about [this](https://github.com/Palmatum/hardhat-js-plugin-boilerplate). Like if it's the thing that you wanted, or if it's missing something or maybe it's just completely some other thing. 

Thank you.

**Comment by fvictorio** [comment]
Hey @Palmatum, sorry for not reviewing this yet! We really appreciate your effort, but reviewing things takes a lot of time and we are kind of busy at the moment, so please don't take it personally :smile: We'll try to take a look at this as soon as we can.

**Comment by Palmatum** [comment]
> Hey @Palmatum, sorry for not reviewing this yet! We really appreciate your effort, but reviewing things takes a lot of time and we are kind of busy at the moment, so please don't take it personally 😄 We'll try to take a look at this as soon as we can.

Oh it's all okay. I thought I did something wrong and that's why I didn't get a response.

**Comment by Saty248** [comment]
hi @alcuadrado @fvictorio is this issue solved yet?

**Comment by koyuncuoglum95** [comment]
Hello @alcuadrado @fvictorio, I'd like to contribute this project. I'm here first time.

**Comment by koyuncuoglum95** [comment]
@fvictorio, I understood what you meant. When I'm done, let you know.

**Comment by Saty248** [comment]
@fvictorio @alcuadrado i read the above comment and created this boilerplate accordingly.. please check this repo 

[repo](https://github.com/Saty248/hardhat-js-plugin-boilerplate)

**Comment by Akshat-sGit** [comment]
Hey @fvictorio @alcuadrado, I would like to contribute here. Can you tell me how to?
